packages/xs-vat-worker/test/test-boot-lockdown.js

```diff
@@ -6,7 +6,10 @@ import * as path from 'path';
 import { xsnap } from '@agoric/xsnap';
 
 const dist = async name =>
-  fs.promises.readFile(path.join(__filename, '..', '..', 'dist', name));
+  fs.promises.readFile(
+    path.join(__filename, '..', '..', 'dist', name),
```

Comment:
Using `__filename` creates coupling to `node -r esm` (“RESM”) since Node.js’s and the web’s ESM loader does not provide this global. They provide `import.meta.url` instead. Unfortunately, RESM doesn’t provide `import.meta.url`, so there’s no way to write modules that straddle compatibility.

My recommendation is to pre-pay some migration debt by capturing `const importModuleUrl = `file://${__filename}` up top and write everything else in terms of that constant. Then, eventually, we can `sed` our way to proper ESM support later.